app/controllers/api/v1/api_keys_controller.rb

```diff
@@ -7,9 +7,9 @@ def show
       sign_in User.authenticate(username, password)
       if current_user
         respond_to do |format|
-          format.any(:all) { render text: current_user.api_key }
+          format.any(:all) { render plain: current_user.api_key }
```

Comment:
Replacing `render text` with `render plain` seems like an unnecessary deprecation warning fix? Should be a follow-up PR.